posthog/api/utils.py

```diff
@@ -281,3 +282,19 @@ def get_event_ingestion_context_for_personal_api_key(
         return EventIngestionContext(team_id=team_id, anonymize_ips=anonymize_ips)
     except Team.DoesNotExist:
         return None
+
+
+def check_definition_ids_inclusion_field_sql(raw_included_definition_ids: Optional[str], is_property: bool):
+    included_definition_ids: Tuple = ()
+    if raw_included_definition_ids:
+        included_definition_ids = json.loads(raw_included_definition_ids)
+    included_definition_ids = tuple(set(included_definition_ids))
+
+    # Create conditional field based on whether id exists in included_properties
+    included_definitions_sql = f"{{{','.join(included_definition_ids)}}}"
```

Comment:
Are you able to make this use [named parameters](https://www.psycopg.org/psycopg3/docs/basic/params.html)? It seems this is parsed directly from the request which could be easy target for SQL injection

Reply:
ah good catch, sure can do